Misc/NEWS.d/next/Library/2024-12-27-16-28-57.gh-issue-128302.2GMvyl.rst

```diff
@@ -0,0 +1,4 @@
+Allow :meth:`!xml.dom.xmlbuilder.DOMParser.parse` to correctly handle
```

Comment:
Can you create two different NEWS entries for the resolveEntity fix please?

Reply:
Done.